site-src/guides/metrics.md

```diff
@@ -93,4 +93,73 @@ TOKEN=$(kubectl -n default get secret inference-gateway-sa-metrics-reader-secret
 kubectl -n default port-forward inference-gateway-ext-proc-pod-name  9090
 
 curl -H "Authorization: Bearer $TOKEN" localhost:9090/metrics
-```
\ No newline at end of file
+```
+
+## Prometheus Alerts
+
+The section instructs how to configure prometheus alerts using collected metrics.
+
+### Configure alerts
+
+You can follow this [blog post](https://grafana.com/blog/2020/02/25/step-by-step-guide-to-setting-up-prometheus-alertmanager-with-slack-pagerduty-and-gmail/) for instruction of setting up alerts in your monitoring stacks with Prometheus.
+
+The alert template in this section follows the [Template examples](https://prometheus.io/docs/prometheus/latest/configuration/template_examples/).
```

Comment:
> The alert template in this section

Are you referring to `tools/alerts/alter.yaml`? If so, please link to this file.

Reply:
Updated.